packages/storage-plus/src/de_old.rs

```diff
@@ -0,0 +1,106 @@
+use std::array::TryFromSliceError;
+use std::convert::TryInto;
+
+use cosmwasm_std::{StdError, StdResult};
+
+use crate::de::KeyDeserialize;
+use crate::keys_old::IntKeyOld;
```

Comment:
IntKeyOld. I love it

Reply:
Yeah. It can be useful for people that are using signed int keys and don't want to migrate, by the way.

We can keep these around for a while, and then delete / clean up the whole thing.